src/actions/tango.js

```diff
@@ -44,6 +44,7 @@ export function executeCommand(command, argin, device) {
     try {
       const result = await TangoAPI.executeCommand(command, argin, device);
       dispatch({ type: EXECUTE_COMMAND_COMPLETE, command, result, device });
+      console.log(result);
```

Comment:
This is possibly an artefact of debugging but this application uses redux-logger and sends any redux actions to the console anyway. I'd suggest removing this to avoid log clutter.

Reply:
fixed